src/commons/api/roas.rs

```diff
@@ -324,27 +340,27 @@ impl RoaDefinitionUpdates {
     }
 
     pub fn add(&mut self, add: RoaDefinition) {
-        self.added.insert(add);
+        self.added.push(add);
     }
 
-    pub fn added(&self) -> &HashSet<RoaDefinition> {
+    pub fn added(&self) -> &Vec<RoaDefinition> {
         &self.added
     }
 
-    pub fn removed(&self) -> &HashSet<RoaDefinition> {
+    pub fn removed(&self) -> &Vec<RoaDefinition> {
         &self.removed
     }
 
     pub fn remove(&mut self, rem: RoaDefinition) {
-        self.removed.insert(rem);
+        self.removed.push(rem);
```

Comment:
Now that this is a Vec instead of a HashSet should this reject duplicates to maintain previous behaviour?